react/features/shared-video/middleware.native.js

```diff
@@ -70,7 +70,7 @@ StateListenerRegistry.register(
                     const localParticipantId = getLocalParticipant(getState()).id;
                     const status = attributes.state;
 
-                    if ([ 'playing', 'pause', 'start' ].includes(status)) {
+                    if (isSharingStatus(status)) {
```

Comment:
Here I thought it would be better and cleaner if we use the helper function.

Reply:
👍